sns/parity_math.c

```diff
@@ -297,12 +346,16 @@ M0_INTERNAL void m0_parity_math_fini(struct m0_parity_math *math)
 		m0_matvec_fini(&math->pmi_sys_res);
 //		m0_parity_fini();
 	}
+#endif /* __KERNEL__ */
 }
 
 M0_INTERNAL int m0_parity_math_init(struct m0_parity_math *math,
 				    uint32_t data_count, uint32_t parity_count)
 {
 	int ret;
+#ifndef __KERNEL__
+	uint32_t total_count;
+#endif /* __KERNEL__ */
```

Comment:
We can define this variable in following section where it is used.

Usually we define a variable at the beginning of a function. But for simplicity, I think it is OK to do in the other way.

Reply:
Done.